server/models/Conversation.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
+  {
+    sender: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    receiver: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    description: {
+      type: String,
+      default: "",
+    },
```

Comment:
```suggestion
    },
    read: {
      type: Boolean,
      required: true,
      default: false,
    }
```
Why don't we add a `read` status to each message

Reply:
So I will add this, but does that mean I have to add a new POST route that will edit read to true when the receiver clicks on the conversation in the frontend? On the frontend,  useEffect will render all conversations with getAllConversations GET request, then user will click on a conversation tab on the left of the dashboard, it will render that conversation on the chatbox to the right, then we will make a POST request to mark any unread messages as read? That is how I am imagining this, is this correct?

Reply:
@rj13371 Yes, this is correct. It's the same functionality we discussed in our meeting on Friday